code/game/jobs/job/medical.dm

```diff
@@ -109,6 +109,55 @@
 	icon_state = "player-green"
 	join_tag = /datum/job/doctor
 
+/datum/job/medstudent
+	title = "Soteria Med Student"
```

Comment:
```suggestion
	title = "Soteria Medical Resident"
```

1) We shouldn't be abbreviating like that in job titles
2) Resident is a better title for this role